### ISSUE TITLE
"http://www.fact-bounty.org/" in the project description is not working

### ISSUE THREAD

**Issue by Raj2367** [scorelab/fact-Bounty#485]
# Prerequisites

- [ ] I am running the latest version
- [ ] I checked to make sure that this issue has not already been filed

# Expected Behavior

"http://www.fact-bounty.org/" should directed to the website

# Current Behavior

It is showing an error.

**Comment by hv7214** [comment]
I guess website might be down for maintenance.

**Comment by kush-daga** [comment]
It isn't working yet, is it down for maintenance?

**Comment by Anmolbansal1** [comment]
Yes, it is down currently and will be deployed with latest version soon :smile:
